core/templates/pages/skill-editor-page/editor-tab/skill-prerequisite-skills-editor/skill-prerequisite-skills-editor.component.ts

```diff
@@ -78,6 +78,12 @@ implements OnInit {
     const sortedSkillSummaries = this.groupedSkillSummaries.current.concat(
       this.groupedSkillSummaries.others);
     const allowSkillsFromOtherTopics = true;
+    const skillIdsToExclude = {
+      [this.skill.getId()]: true
+    };
+    for (let idx in this.skill.getPrerequisiteSkillIds()) {
```

Comment:
Can you find a way to rewrite this bit of code without using `idx` so that it is easier to follow?

Reply:
I looked at other parts of this file and found this way of getting the prerequisite skill ids of a skill. But how should I do it differently here? Just change the idx name to something else or change the way we get the prerequisite skills to something like this:

```
this.skill.getPrerequisiteSkillIds().forEach(prerequisiteSkillId => {
    skillIdsToExclude[prerequisiteSkillId] = true;
});
```

Reply:
Yes, something like what you propose here is reasonable (i.e. getting rid of idx entirely since you don't really need it).

Reply:
Done